students/tammyd/lesson03/mailroom_pt1.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+"""
+
+Mailroom program for a non profit
+
+"""
+
+import sys
+
+donor_db = [("Britney Spears", [50.99, 100.99, 2000.99]),
+            ("Christina Aguilera", [50.99, 100.99, 2000.99]),
+            ("Mandy Moore", [50.99, 100.99, 2000.99]),
+            ("Jessica Simpson", [50.99, 100.99, 2000.99]),
+            ("Queen Bey", [50.99, 100.99, 2000.99]),
+            ]
+
+def mainloop():
+    print("Welcome to the Mailroom \n")
+
+def main_menu():
+    print("Main Menu:")
+    print(
+      "T - Send a thank you\n"
+      "R - Create a report\n"
+      "Q - Quit the program"
+      )
+    menu_selection = input("\nChoose a letter:  ")
+    if menu_selection.upper().strip() == "T":
+        thank_you()
+    elif menu_selection.upper().strip() == "R":
+        create_report()
+    elif menu_selection.upper().strip() == "Q":
```

Comment:
I like that you used .upper().strip()! A slightly more efficient way would be to put those methods on menu_selection before comparing them. Like:
```
menu_selection = input("select stuff!")
menu_selection = menu_selection.upper().strip()
if menu_selection == "T":
#etc. etc.
```

Also, in the future we can use dicts instead of a bunch of "elif"s.